public/css/application.css

```diff
@@ -0,0 +1,83 @@
+body {
+  counter-reset:section;
+  background-color: #F6F6EF;
+  font-family: Verdana;
+  max-width: 700px;
+  margin: 0 auto;
+}
+
+.container {
+  margin: 20px auto;
+  max-width: 600px;
+  padding: 5px 20px 20px 20px;
+  background-color: #fff;
+  /*border: 1px dotted black;*/
+}
+
+.logged-in-title {
+  margin-bottom: 0;
+}
+
+.new-list-link {
+  margin-top: 0;
+}
+
+.errors {
+  color: red;
+}
+
+article {
```

Comment:
Try to use classes and IDs vs actual elements; that way, new things added to the page only get the styling you deliberately want them to have.